core/schema/src/main/resources/org/hawkular/schema/cassalog.groovy

```diff
@@ -20,4 +20,5 @@ include '/org/hawkular/schema/bootstrap.groovy'
 
 setKeyspace keyspace
 
-include '/org/hawkular/schema/updates/schema-0.15.0.groovy'
\ No newline at end of file
+include '/org/hawkular/schema/updates/schema-0.15.0.groovy'
+include '/org/hawkular/schema/updates/schema-0.16.0.groovy'
```

Comment:
The versions and the file names are completely arbitrary as far as Cassalog is concerned. The `include` directive is telling Cassalog which scripts to execute. It just so happens that we have adopted a convention to have a separate script per release (when there are schema changes).